editor/plugins/script_editor_plugin.cpp

```diff
@@ -4074,17 +4074,14 @@ ScriptEditor::ScriptEditor(WindowWrapper *p_wrapper) {
 	script_forward->set_disabled(true);
 	script_forward->set_tooltip_text(TTR("Go to next edited document."));
 
-	if (p_wrapper->is_window_available()) {
-		menu_hb->add_child(memnew(VSeparator));
+	menu_hb->add_child(memnew(VSeparator));
 
-		make_floating = memnew(ScreenSelect);
-		make_floating->set_flat(true);
-		make_floating->set_tooltip_text(TTR("Make the script editor floating."));
```

Comment:
Seems like this tooltip is lost and now it refers to script/shader editor as "this panel". Since the disabled logic runs in the constructor, you could check if the button is not disabled and ovewrite the tooltip.